styles/style.css

```diff
@@ -0,0 +1,680 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    font-family: sans-serif;
+    font-size: 1.3rem;
+}
+
+header {
+    display: flex;
+    flex-direction: row;
+    background-image: url(../images/header-image.jpg);
+    background-repeat: no-repeat;
+    background-size: cover; 
+    width: 100%;
+    height: 375px;
+}
+
+.top-left {
+    display: flex;
+    flex-direction: column;
+    padding: 4rem 0 0 8rem;
+}
+
+div.checkout {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    padding: 4rem 0;
+}
+
+.top-logo {
+    max-width: 100%; 
+    width: 100%;
+}
+
+.slogan {
+    font-style: italic;
+    font-size: 1.7rem;
+    color: #a1c349;
+    margin-top: 0;
+    width: fit-content;
+}
+
+.slogan p {
+    margin: 0;
+}
+
+[src="./images/cart-icon.svg"] {
+    max-width: 100%;
+    width: 5%;
+}
+
+.total {
+    color: #FFF;
+    padding-bottom: 0;
+}
+
+[value="Checkout"] {
+    padding: 0.5rem 2rem;
+    margin-top: 1rem;
+    background-color: #a1c349;
+    color: #FFFfff;

```

Comment:
make sure all of these leters are the same case. I prefer all lowercase `#ffffff`

Reply:
Thanks a lot for letting me know!!
I think that it is a good practice to specify hex code in lowercase,  But my question is, in general, that makes the code more consistent to avoid some errors on the page? or even is it visually better to other users??  
I have just checked and turned everything in lowercase.
Thanks again. I appreciate your help!!!